docs/docs/etl-pipeline-tutorial/7-visualize-data.md

```diff
@@ -29,6 +29,12 @@ There will now be a directory `dashboard` within the root of the project.
 └── uv.lock
 ```
```

Comment:
I recommend changing the title of this page to something like "Build a dashboard to visualize data" (you can leave the filename the same)

Reply:
I would edit the intro paragraph a little:

```
In this step, we will visualize some of the data we have been modeling in a dashboard using [Evidence](https://evidence.dev/) connected to our model assets.
```

Reply:
I would rewrite step 1 as follows:

```
First, we will clone an Evidence project that is already configured to work with the data we have modeled with dbt:
```